feed/admin.py

```diff
@@ -9,5 +10,14 @@ class PostAdmin(admin.ModelAdmin):
     list_filter = ("status",)
     search_fields = ['title', 'content']
     prepopulated_fields = {'slug': ('title',)}
+
+class CommentAdmin(admin.ModelAdmin):
```

Comment:
CommentAdmin is being added here, but it's not being registered so this code isn't actually being used.